### PR TITLE
Implement navigation

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/Home/styles.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.main {
+  display: flex;
+  height: 70vh;
+  border-bottom: 2px solid gray;
```
Comment (mi-chal) [inline src/screens/Home/styles.scss]: Keep colours in 1 place.

File: src/screens/Home/styles.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  position: absolute;
+  padding-top: 40vh;
+}
+
+.main__button {
+  display: flex;
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  background-color: lightskyblue;
```
Comment (mi-chal) [inline src/screens/Home/styles.scss]: colors

File: src/screens/Home/styles.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  align-items: center;
+  padding-left: 5px;
+  font-size: 1.6rem;
+  color: white;
+  border: none;
+  cursor: pointer;
+  outline: 0;
+  &:hover {
+    background-color: red;
+    color: black;
```
Comment (mi-chal) [inline src/screens/Home/styles.scss]: colors

File: src/shared/components/Main/Main.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import MyBooking from 'screens/MyBooking/MyBooking';
+import TrackingFlights from 'screens/TrackingFlights/TrackingFlights';
+import Home from 'screens/Home/Home';
+
+import './styles.scss';
+
+function Main() {
+  return (
+    <Switch>
+      <Route path="/Home" exact component={Home} />
```
Comment (mi-chal) [inline src/shared/components/Main/Main.jsx]: Route paths should not be capitalized

Comment (dwantula) [reply src/shared/components/Main/Main.jsx]: Fixed

File: src/screens/Home/styles.scss (hunk 2 above)
Comment (dwantula) [reply src/screens/Home/styles.scss]: Done

File: src/screens/Home/styles.scss (hunk 1 above)
Comment (dwantula) [reply src/screens/Home/styles.scss]: Done

File: src/screens/Home/styles.scss (hunk 3 above)
Comment (dwantula) [reply src/screens/Home/styles.scss]: Done

**Review by mi-chal — APPROVED** [review]

**Pull request merged**